trace-sdk/src/main/java/io/bitrise/trace/session/ApplicationSessionManager.java

```diff
@@ -76,4 +77,14 @@ public synchronized void stopSession() {
   public Session getActiveSession() {
     return session;
   }
+
+  @VisibleForTesting
+  static synchronized boolean isSessionManagerActive() {
```

Comment:
Added these two booleans for the tests - so that i could verify that the reset made both things null. Does that make sense?

Reply:
Okay.